src/rprblender/nodes/node_parser.py

```diff
@@ -273,7 +293,9 @@ def final_export(self):
         """
 
         log("export", self.material, self.node, self.socket_out, self.group_nodes)
-        if isinstance(self.rpr_context, RPRContextHybrid):
+        if self.node.mute:
+            return self.export_muted()
```

Comment:
Use node_item = self.export_muted() here, because it is going to be processed further in this function.